vagrant/provisioning/roles/haproxy/tasks/main.yml

```diff
@@ -447,13 +447,16 @@
     force: yes
     backup: yes
 
-- name: Change haproxy service name
+- name: Ensure that haproxy service file exists
```

Comment:
I think something like "Check whether the haproxy service file exists" would better reflect what this task is doing